docker/vernemq/bin/vernemq.sh

```
@@ -0,0 +1,352 @@
+IP_ADDRESS=${DOCKER_IP_ADDRESS:-${IP_ADDRESS}}
+
+VERNEMQ_ETC_DIR="/vernemq/etc"
+VERNEMQ_VM_ARGS_FILE="${VERNEMQ_ETC_DIR}/vm.args"
+VERNEMQ_CONF_FILE="${VERNEMQ_ETC_DIR}/vernemq.conf"
+VERNEMQ_CONF_LOCAL_FILE="${VERNEMQ_ETC_DIR}/vernemq.conf.local"
+
+SECRETS_KUBERNETES_DIR="/var/run/secrets/kubernetes.io/serviceaccount"
+
+# Function to check istio readiness
+istio_health() {
+  cmd=$(curl -s http://localhost:15021/healthz/ready > /dev/null)
```

Shellcheck (0.10.0)

[warning] cmd appears unused. Verify use (or export if used externally).

(SC2034)